accounting/receivables/customer_invoices/payment_terms.rst

```diff
@@ -1,96 +1,109 @@
-==================================
-How to setup and use payment terms
-==================================
+===================================
+Payment Terms and Installment Plans
+===================================
 
-Payment terms define the conditions to pay an invoice. They apply on
-both customer invoices and supplier bills.
+**Payment Terms** regroup all the conditions under which a sale is completed and paid. They can be
+applied to sales orders, customer invoices, and supplier bills, mostly to ensure that they will be
+correctly paid, and in time. These conditions cover:
 
-Example, for a specific invoice:
+- The due date
+- Some discounts
+- Any other condition on the payment
 
--  Pay 50% within 10 days
+Defining Payment Terms automates the computation of payments due dates, both for invoices and bills.
+This is particularly helpful in managing installment plans.
 
--  Pay the remaining balance within 30 days
+An **installment plan** allows the customers to pay an invoice in parts, with the amounts and
+payment dates defined beforehand by the seller.
 
-.. note::   Payment terms are different from invoicing in several areas. If,
-            for a specific order, you invoice the customer in two parts, that's not
-            a payment term but invoice conditions.
+**Examples of Payment Terms:**
 
-Configuration
-=============
-
-Configure your usual payment terms from the Configuration menu of the
-Account application. The description of the payment term is the one that
-appear on the invoice or the sale order.
-
-A payment term may have one line (ex: 21 days) or several lines (10%
-within 3 days and the balance within 21 days). If you create a payment
-term with several lines, be sure the latest one is the balance. (avoid
-doing 50% in 10 days and 50% in 21 days because, with the rounding, it
-may not do exactly 100%)
-
-.. todo:: screenshot payment term forms, after QDP have commited the change
-          planned on this object
-
-Using Payment Terms
-===================
+- | **Immediate Payment**
+  | The full payment is due on the day of the invoice's issuance.
+- | **15 Days** (or **Net 15**)
+  | The full payment is due 15 days after the invoice date.
+- | **21 MFI**
+  | The full payment is due by the 21st of the month following the invoice date.
+- |	**2% 10, Net 30 EOM**
```

Comment:
```suggestion
- |  **2% 10, Net 30 EOM**
```

Reply:
Weird!
I couldn't see it with PyCharm. Thanks 🌈